docs/design/FeatureSwitches.md

```diff
@@ -26,3 +27,26 @@ When this feature is disabled, any value set to [`SearchHandler.DisplayMemberNam
 ## MauiQueryPropertyAttributeSupport
 
 When disabled, the `[QueryProperty(...)]` attributes won't be used to set values to properties when navigating. Instead, implement the [`IQueryAttributable`](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/navigation#process-navigation-data-using-a-single-method) interface whenever you need to accept query parameters.
+
+## MauiImplicitCastOperatorsUsageViaReflectionSupport
+
+When disabled, MAUI won't look for implicit cast operators when converting values from one type to another. This can affact the following scenarios:
+- bindings between properties with different types,
+- setting a property value of a bindable object with a value of different type.
+
+If your library or your app defines an implicit operator on a type that can be used in one of the previous scenarios, you should define a custom `TypeConverter` for your type and attach it to the type using the `[TypeConverter(typeof(MyTypeConverter))]` attribute.
+
+If you need to define a type converter for a type that you don't own, you can use `MauiAppBuilder.ConfigureTypeConversions`:
+
+```c#
+var builder = MauiApp.CreateBuilder();
+builder
+    .UseMauiApp<App>()
+    .ConfigureTypeConversions(config =>
+    {
+        config.AddTypeConverter<MyTypeA, MyTypeAConverter>();
```

Comment:
Could `MyTypeA` be a simple POCO example type or record like `Customer` or `Animal`?

Then could we show what `MyTypeAConverter` (`CustomerConverter`) looks like below?

Reply:
Yes, sure. I will extend the sample.